packages/strapi-plugin-users-permissions/services/Providers.js

```diff
@@ -286,25 +267,9 @@ const getProfile = async (provider, query, callback) => {
       break;
     }
     case 'microsoft': {
-      const microsoft = new Purest({
+      const microsoft = purest({
```

Comment:
Question, is this going to change the way the `Microsoft` authentication works?

Reply:
As far as my testing with other OAuth providers - Discord, Twitter, Google - it doesn't appear so, but as I mentioned in the PR I could really use a hand testing other integrations.

If you have the time - and MS developer access to set up a test project - you can follow the instructions [here](https://github.com/strapi/strapi-examples/tree/master/login-react) to test this specific integration.

Its pretty quick to get this up an running, the only problem you may run into is that MS (or other service providers, such as Google) do not like `localhost`. 

If you have a chance to follow up on this and you run into any problems, I am on the community slack under this username and can save you a couple searches to figure out how to resolve that problem.

Reply:
I'll message you on Slack, I'll pull the latest update and test it out with Microsoft

Reply:
Per @wdhodges latest comment (below) I'll resolve this convo since we know MS provider works :)